modules/templates/conf.py_t

```diff
@@ -71,7 +70,8 @@ extensions = ['sphinx.ext.githubpages',
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
     'javasphinx',
-    'sphinxcontrib.apiblueprint']
+    'sphinxcontrib.apiblueprint',
+    'timeline']
```

Comment:
Here also, `extensions.timeline` should work without needing to add extensions to sys.path. Please confirm and if true, make the changes.